robot_ws/src/cloudwatch_robot/package.xml

```diff
@@ -35,4 +35,6 @@
 
   <!-- Turtlebot -->
   <depend>turtlebot3_msgs</depend>
+  <!-- Required to install .rviz model-->
+  <depend>turtlebot3_description</depend>
```

Comment:
This hasn't been changed to `turtlebot3_description_reduced_mesh`.

Reply:
turtlebot3_description is needed for the rviz, so we still need it. And yes, we can add turtlebot3_description_reduced_mesh as dependency as well, and do the same for hello world sample app.

Reply:
@mm318 Added `turtlebot3_description_reduced_mesh` as a dependency. Please take a look.